docs/howto/cancel-running-job.rst

```diff
@@ -0,0 +1,29 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+
+How To Cancel a Running Job
+=========================
+
+The task can be marked as success or not failed from the web interface. When the task changes state it will be automatically stopped.
```

Comment:
"automatically stopped" might be a bit unclear to some people.
What actually happens is that Airflow executes the `on_kill` method of the operator.
This is very relevant for cases where the operator executes a job on a remote machine.
User needs to understand that if the operator did not implement `on_kill` then clearing the Airflow task is not enough the user must also kill the process in the remote machine.